app/components/reviews-dashboard/template.hbs

```diff
@@ -0,0 +1,23 @@
+<div class='reviews-dashboard content'>
+    <div class='reviews-dashboard-header'>
+        <div class='container'>
+            <div class='row'>
+                <div class='text-center m-t-xl col-xs-12'>
+                    <h1>{{t 'dashboard.title'}}</h1>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class='container'>
+        <div class='row'>
+            <div class='col-md-4 col-md-push-8'>
+                {{dashboard-sidebar providers=providers}}
+            </div>
+            <div class='col-md-8 col-md-pull-4'>
+                {{action-feed
+                    toDetail=(action transitionToDetail)
```

Comment:
I was trying to avoid that since it seems very anti-Ember but we are ignoring pretty much everything they recommend so I guess it doesn't really matter. I also hate passing things down 10 levels.

Reply:
I think it'd make more sense if these components were meant to be reusable outside this app, or the action varied depending on other state, or it did anything other than a simple transition.

They added `router`, though, so I figure we may as well use it.

Reply:
@aaxelb, bad news. We can't use the router service yet because their is a [bug](https://github.com/emberjs/ember.js/pull/15613) that breaks when all query params aren't present. So when we try and use `this.get('router').transitionTo(...)` we always get an error about the [ticket query parameter on the application route](https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/mixins/osf-cookie-login-controller.js#L20).
```
"Assertion Failed: You passed the `false` query parameter during a transition into application, please update to ticket"
``` 

Fix release:
> Apologies for not getting this in for 2.16, I'll get the changes into the next 2.17 beta release and once some folks can confirm all is working well for us I'll pull down into 2.16 for the LTS.

Reply:
That fix is in [2.16.1](https://github.com/emberjs/ember.js/releases/tag/v2.16.1)... we should upgrade. In another ticket.

Reply:
Oh nice, I missed that. I'll add a ticket 👍